app/experimenter/experiments/tasks.py

```diff
@@ -110,3 +112,56 @@ def add_experiment_comment_task(user_id, experiment_id):
         metrics.incr("add_experiment_comment.failed")
         logger.info("Failed to create bugzilla comment")
         raise e
+
+
+@app.task
+@metrics.timer_decorator("update_experiment_status.timing")
+def update_experiment_status():
+    metrics.incr("update_experiment_status.started")
+    accepted_experiments = Experiment.objects.filter(
+        Q(status=Experiment.STATUS_ACCEPTED) | Q(status=Experiment.STATUS_LIVE)
+    )
+    status_mapping = update_mapping()
+    for experiment in accepted_experiments:
+        try:
```

Comment:
We should emit a log msg here saying something like 'Updating Experiment Whatever'